### ISSUE TITLE
Installing can't be done with the script

### ISSUE THREAD

**Issue by Cr0a3** [rust-lang/rustup#3743]
### Problem

Hi, 
I want to use Rust in GitHub-Codespaces. But when I run the script:
`curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
I get following error:
`error: could not amend shell profile: '/home/codespace/.config/fish/conf.d/rustup.fish': could not write rcfile file: '/home/codespace/.config/fish/conf.d/rustup.fish': No such file or directory (os error 2)`
But it installes `rustup`. Then you need to restart your shell and run: `rustup install stable`
I hope you can fix it for GitHub codespaces.

### Steps

Run the installation script: `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`

### Possible Solution(s)

Descripted in the Problem:
But it installes `rustup`. Then you need to restart your shell and run: `rustup install stable`
I hope you can fix it for GitHub codespaces.

### Rustup version

```console
rustup 1.27.0 (bbb9276d2 2024-03-08)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.77.0 (aedd173a2 2024-03-17)`
```

### Installed toolchains

```console
No toolchain is preinstalled. You need to do it manualy
```

**Comment by djc** [comment]
I think this might have been fixed by #3712?

**Comment by rami3l** [comment]
@djc Yes, it's a duplicate of #3706.
